gen-web/index.html

```diff
@@ -1,35 +1,255 @@
+                    <h4 id="timer-sub-text" class="text-white subheader-text-bookend"></h4>
+                  </div>
+                </div>
+                <div class="stepLabel step5">
+                  <p class="subheader-text-bookend text-white"> Once your HoloPort is connected to the Holo network you will receive an email with a link to your HoloPort admin panel.</p>
+                </div>
+                <div class="stepLabel step5 block-bottom-text">
+                  <p class='subheader-text-bookend text-white'>Note: If you don’t receive an email within 5 minutes, please reach out to <a href="https://forum.holo.host" target="_blank" class="text-white underline">customer service</a>.</p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="stepButton step0">
+            <div class="break-text">
+              <button id="start-button" class="btn first-btn button-font"><span>Next</span></button>
+              <p class='note-text enter-text text-white'>or press Enter</p>
+            </div>
+          </div>
+        </div>
+        <img id="holoport-flying-bookend" class="stepLabel step0 step5" src='assets/images/hports-flying-transparent.png' />
+      </main>
+      <main id='content-container' class="stepHeader step1 step2 step3 step4">
+        <div class="content-wrapper">
+          <div class="stepHeader step1 step2 step3 step4 card-wrapper">
+            <div class="card-transparent left-margin">
+              <div class="stepLabel step1 step2 step3 step4">
+                <ul class="progressbar">
+                  <li class="progressbar-item 1 active"></li>
+                  <li class="progressbar-item 2"></li>
+                  <li class="progressbar-item 3"></li>
+                  <li class="progressbar-item 4"></li>
+                </ul>
+
+                <div class="stepLabel step1 pageHeader">
+                    <div class="step-label-info"><span class="holo-green">Step 1.</span> Register your HoloPort by making login credentials.</div>
+                    <span id="form-error-message" class="error-red"></span>
+                    <span id="form-success-message" class="holo-green"></span>
+                </div>
+                <div class="stepLabel step2 pageHeader">
+                    <div class="step-label-info"><span class="holo-green">Step 2.</span> Download the configuration file, then, drag file from your downloads folder to your USB drive.</div>
+                </div>
+                <div class="stepLabel step3 pageHeader">
+                    <div class="step-label-info"><span class="holo-green">Step 3.</span> Insert USB into your HoloPort to transfer the configuration file.</div>
+                </div>
+                <div class="stepLabel step4 pageHeader">
+                    <div class="step-label-info"><span class="holo-green">Step 4:</span> Connect your HoloPort to the Holo network.</div>
+                </div>
```

Comment:
@alastairong @peeech @yegortimoshenko  - It looks like we're now missing the field for 'device-name'.  Can we make sure this is added back?!

Also, can we please following the same process as @peeech when suggesting updates to the UI and create a PR for any updates.   We need to maintain a clear way of tracking work and processing updates to avoid implementing unapproved updates to changes in decision decisions - such as this missing feild.

Reply:
Right, I'm sorry. I just saw this.